fury/tests/test_actors.py

```diff
@@ -756,27 +760,45 @@ def test_dots(interactive=False):
 
     npt.assert_equal(scene.GetActors().GetNumberOfItems(), 1)
 
-    extent = scene.GetActors().GetLastActor().GetBounds()
-    npt.assert_equal(extent, (0.0, 1.0, 0.0, 1.0, 0.0, 0.0))
+    arr = window.snapshot(scene)
+    report = window.analyze_snapshot(arr, colors=colors)
+    npt.assert_equal(report.objects, 3)
+    npt.assert_equal(report.colors_found, [False]*3)
+
+    # Test three points with one color and opacity
+    points = np.array([[1, 0, 0], [0, 0, 1], [0, 1, 0]])
+    colors = (1, 0, 0)
+    dot_actor = actor.dot(points, colors=colors, opacity=.5)
+
+    scene.clear()
+    scene.add(dot_actor)
+    scene.reset_camera()
+    scene.reset_clipping_range()
+
+    if interactive:
+        window.show(scene, reset_camera=False)
 
     arr = window.snapshot(scene)
-    report = window.analyze_snapshot(arr,
-                                     colors=(0, 255, 0))
+    report = window.analyze_snapshot(arr, colors=colors)
     npt.assert_equal(report.objects, 3)
+    npt.assert_equal(report.colors_found, [False]*1)
```

Comment:
Same as above, make sure these tests always return true. Use opacity to calculate the appropriate percentage of 255.